test/named_array_partition_tests.jl

```diff
@@ -0,0 +1,30 @@
+@testset "NamedArrayPartition tests" begin
+    x = NamedArrayPartition(a = ones(10), b = rand(20))
```

Comment:
missing `using RecursiveArrayTools, Test` on top

Reply:
Shoot - thank you for catching that.